### DISCUSSION TITLE
Monorepos

### DISCUSSION THREAD

**Discussion by lonix1** [olivierlacan/keep-a-changelog#510] [Q&A]
Maintaining changelogs for monorepos is tricky.

Some info about that would be useful.

**Comment by recfab** [comment]
I am interested in this as well.

I'm trying to push us towards a mono repo at work, and also want to adopt a changelog.

My first thought was something like:

> # Unreleased
> 
> ## ComponentA
> 
> * Did something
> * Did another thing
> 
> ## Component B
> * Something else
> * And even more
> 
> ## ComponentB, ComponentC
> * Something that affects multiple components

**Comment by martins-1992** [comment]
I personally create one changelog per project in the mono repo and have a program that merges all changelogs into one for rendering.
